ayekan/opensearch/clusters/logging.yaml

```diff
@@ -0,0 +1,350 @@
+---
+apiVersion: opensearch.opster.io/v1
+kind: OpenSearchCluster
```

Comment:
Was it not possible to use the https://github.com/opensearch-project/opensearch-k8s-operator/tree/main/charts/opensearch-cluster chart?

Reply:
It's certainly possible to use the helm chart, but since the helm chart sole resource IS the OpensearchCluster object and I needed to ship some additional components (user, ingress, opensearch rbac, ...) with it, I found it easier to just glob everything in kubernetes objects for the single cluster. Opposed to having to deploy a helm chart and the additional resources.

Reply:
Fair enough. Lets merge this as-is and I'll start working on migrating the deployment to fleet.